apisix/plugins/ext-plugin/init.lua

```diff
@@ -612,10 +612,27 @@ local rpc_handlers = {
             end
 
             local len = rewrite:RespHeadersLength()
+            local excludeRespHeader = {
```

Comment:
We can just use a hash-like to save these headers so we don't need to iterate all of them, i.e. we can reduce the time complicity from O(N^2) => O(N).